app/connector/support/processor/src/test/java/io/syndesis/connector/support/processor/ErrorMapperTest.java

```diff
@@ -60,4 +66,13 @@ public void testUnmappedException() {
         Assert.assertEquals(ErrorCategory.SERVER_ERROR   ,info.getCategory());
         Assert.assertEquals("Unmapped Exception"         ,info.getMessage());
     }
+
+    @Test
+    public void testFileAlreadyExistsException() {
+        Exception e = new FileAlreadyExistsException("myfile");
+        ErrorStatusInfo info = ErrorMapper.mapError(e, errorResponseCodeMappings, 200);
+        Assert.assertEquals(Integer.valueOf(409)         ,info.getResponseCode());
```

Comment:
Please use AssertJ, the assertions from JUnit/Hamcrest generate failures that are hard to reason about and the code is much less readable, and we're using AssertJ throughout the codebase.

```suggestion
        assertThat(info.getResponseCode()).isEqualTo(409);
```

Reply:
agreed